Detectors/EMCAL/workflow/src/ClusterizerSpec.cxx

```diff
@@ -62,17 +68,34 @@ void ClusterizerSpec<InputType>::run(framework::ProcessingContext& ctx)
   auto Inputs = ctx.inputs().get<std::vector<InputType>>(inputname.c_str());
```

Comment:
this is a vector of PODs, right? You shoud use here ``gsl::span<InputType>`` to avoid copying the data.

Reply:
@hahassan7 Indeed since #3035 the EMCAL publisher supports handling of gsl::span. Please change accordingly.